original/v1/s001-install/instalace.html

```diff
@@ -63,10 +63,29 @@ <h3 class="text-muted">PyLadies Brno</h3>
 
                     <h3>macOS</h3>
                     <p>
-                        Drž se instrukcí pro Windows, ale stáhni instalátor pro <em>Mac OS X</em>,
-                        podle verze tvého systému.
+                        Nainstalujeme si nástroj homebrew. <a href="http://brew.sh">Homebrew</a> za nás řeší a zjednodušuje instalaci aplikací a knihoven, které budeme potřebovat pro programování.
                     </p>
 
+                    <p>
+                        Spusť v příkazové řádce příkaz:
+                    </p>
+
+<pre>
+$ <b>/usr/bin/ruby -e "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install)"</b>
+</pre>
+
+                    <div class="note">
+                        Instalace gitu při instalaci homebrew by si měla sama vyžádat command line developer tools. Pokud se tak nestane stačí zadat xcode-select --install a nainstalovat je ručně.
```

Comment:
Sem bych Git nemíchal, teď instalujeme Python. To, že si Git Homebrew vezme s sebou, je vnitřnost, kterou není třeba vytahovat na povrch.

Ještě by mi přišlo lepší mít trochu srozumitelněji popsané ty „command line developer tools“. Snad by stačilo „[…] vyžádat instalaci podpůrných vývojových nástrojů. Pokud se tak nestane, lze je snadno zavést ručně:

```
<pre>
$ xcode-select --install
</pre>
```

“

Počítám, že v takovém případě je potřeba znovu spustit instalaci Homebrew. To bych také zmínil.